apps/src/templates/CollapserIcon.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+
+const collapseIconStyles = {
+  icon: {
+    fontSize: '18px'
+  }
+};
+
+function CollapserIcon({
+  isCollapsed,
+  onClick,
+  collapsedIconClass,
+  expandedIconClass,
+  style
+}) {
+  const iconClass = isCollapsed ? collapsedIconClass : expandedIconClass;
+
+  return (
+    <i
+      id="ui-test-collapser"
+      onClick={onClick}
+      role="button"
+      className={iconClass + ' fa'}
```

Comment:
Newbie question: what does 'fa' stand for?

Reply:
font awesome! https://fontawesome.com/icons?d=gallery